internal/resolution/variable_sources/bundlesanddependencies/bundles_and_dependencies_test.go

```diff
@@ -192,14 +192,14 @@ var _ = Describe("BundlesAndDepsVariableSource", func() {
 		variables, err := bdvs.GetVariables(context.TODO(), mockEntitySource)
 		Expect(err).NotTo(HaveOccurred())
 
-		var bundleVariables []*bundles_and_dependencies.BundleVariable
+		var bundleVariables []*bundlesanddependencies.BundleVariable
 		for _, variable := range variables {
 			switch v := variable.(type) {
-			case *bundles_and_dependencies.BundleVariable:
+			case *bundlesanddependencies.BundleVariable:
 				bundleVariables = append(bundleVariables, v)
 			}
 		}
-		Expect(len(bundleVariables)).To(Equal(12))
+		Expect(bundleVariables).To(HaveLen(12))
 		Expect(bundleVariables).To(WithTransform(CollectBundleVariableIDs, Equal([]string{"bundle-2", "bundle-1", "bundle-15", "bundle-16", "bundle-17", "bundle-9", "bundle-8", "bundle-7", "bundle-5", "bundle-4", "bundle-11", "bundle-10"})))
```

Comment:
nit: do we need to have the length check if we're already checking equality?

Reply:
Good point. I wasn't paying too much attention to the surrounding lines only validated that `ginkgolinter --fix ./...` gave me reasonable fixes.

I'll submit a follow up request to delete this line & update to use `ConsistOf` as you suggested in another comment.